exe/main.cc

```diff
@@ -26,7 +23,17 @@ namespace HW = HorseWhisperer;
 namespace lth_file = leatherman::file_util;
 
 int startAgent(std::vector<std::string> arguments) {
-#ifndef _WIN32
+#ifdef _WIN32
+    try {
+        Util::daemonize();
```

Comment:
I know. I put in evidence here since I refer to it in my global comment.
The point is that we have a different behaviour in POSIX.